downstream/modules/platform/proc-controller-customize-pod-spec.adoc

```diff
@@ -0,0 +1,36 @@
+[id="controller-customize-pod-spec"]
+
+= Customizing the pod specification
+
+{PlatformNameShort} provides a simple default pod specification, however, you can provide a custom YAML or JSON document that overrides the default pod specification. 
+This field uses any custom fields such as `ImagePullSecrets`, that can be "serialized" as valid pod JSON or YAML. 
+A full list of options can be found in the link:https://docs.openshift.com/online/pro/architecture/core_concepts/pods_and_services.html[Pods and Services] section of the OpenShift documentation.
+
+.Procedure
+
+. To customize the pod specification, specify the namespace in the *Pod Spec Override* field by using the toggle to enable and expand the *Pod Spec Override* field.
++
+image::ag-instance-group-customize-cg-pod.png[Customize pod specification]
++
+Click btn:[Save].
+
+You can provide additional customizations, if needed. Click btn:[Expand] to view the entire customization window:
+
+image::ag-instance-group-customize-cg-pod-expanded.png[Customize pod expanded]
+
+[NOTE]
+====
+The image when a job launches is determined by which {ExecEnvShort} is associated with the job. 
+If you associate a container registry credential with the {ExecEnvShort}, then {ControllerName} attempts to make a `ImagePullSecret` to pull the image. 
```

Comment:
I'm not sure if this is correct but does it make sense to say "make an `ImagePullSecret`" as opposed to "make a `ImagePullSecret`? 

Because it's a variable/attribute I wonder should you say an vs a?